persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -5,6 +5,7 @@
     <cache/>
 
    <resultMap id="genePanelResultMap" type="org.cbioportal.model.GenePanel">
+        <result property="internalId" column="internalId"/>       
```

Comment:
This was  fixed at the service layer. We plan to make it prettier later (mixins?)